packages/zoe/src/zoe.chainmail

```diff
@@ -42,60 +47,52 @@ interface ZoeService {
   getInstance(instanceHandle :InstanceHandle) -> (InstanceRecord);
 
   /** 
-   * To redeem an invite, the user must provide a list of payments as well as
-   * their rules for the offer. 
+   * To redeem an invite, the user must provide their rules for the
+   * offer as well as payments to be escrowed by Zoe.
    * 
-   * The rules for the offer are in two parts: `payoutRules` are used 
-   * by Zoe to enforce offer safety, and `exitRule` is used to specify
+   * The rules for the offer are in three parts: `want` and `offer` are used 
+   * by Zoe to enforce offer safety, and `exit` is used to specify
    * the extent of payout liveness that Zoe can guarantee.
-   * `payoutRules` is a list of objects, each with a `kind` property 
-   * (such as 'offerAtMost') and an amount property. The objects in the 
-   * `payoutRules` must be in the same order
-   * as the issuers associated with a smart contract. That is, the
-   * amount in index 0 of `payoutRules` should be an amount for the issuer
-   * in index 0 of the issuers array. `payments` is an array of the
-   * actual payments to be escrowed, following the rules in the
-   * payoutRules. If the payoutRules kind is 'offerAtMost',
-   * then a payment matching the amount is expected. If the payoutRules
-   * kind is 'wantAtLeast' then the payment will be
-   * ignored and should be `undefined`.
+   * `want` and `offer` are objects with roleNames as keys and amounts
+   * as values. `payments` is an object with the rolesNames as keys,
+   * and the values are the actual payments to be escrowed. A payment
+   * is expected for every rule under `offer`.
```

Comment:
add 'even if some of them will be empty.'

Reply:
I don't think any of them will be empty. Why would that be?

Reply:
I was thinking about the payout, rather than the payments record. That should be documented here as the return value.

Isn't it the case that the SeatAndPayout record will include every keyword, and a possibly empty purse for each?

Reply:
Oh gotcha, yes, the payout will currently include every keyword and a payment (possibly empty).